epochtalk/core-pg
# Mod log

Test Procedure:

1) Run db-migrate

2) DELETE FROM roles;

3) See frontend pull request for mod-log
